stan/math/prim.hpp

```diff
@@ -11,4 +11,6 @@
 #include <stan/math/prim/functor.hpp>
 #include <stan/math/prim/prob.hpp>
 
+#include <stan/math/opencl/prim.hpp>
```

Comment:
Should this be ifdefed?

Reply:
The opencl/prim header has an ifdef on its own but I guess it does not hurt us.

Reply:
Done.